docusaurus/docs/themes/transportation/roads.mdx

```diff
@@ -0,0 +1,623 @@
+defined by its [access restrictions](#access-restrictions) as well as
+local rules, norms, and customs. An example of a transition restriction
+implied by an access restriction, if the segment can only be used along
+the `forward` [heading](shape-connectivity#heading), then it is implied
+that no transitions are allowed to any connected segments if travelling
+along the `backward` heading. An example of a transition restriction
+implied by a local rule or norm would be a blanket prohibition on
+U-turns in a given jurisdiction. (The Overture Transportation schema
+does not specify these local rules, norms, and customs, which are left
+to the specific application to resolve.)
+
+Overture takes a permissive-by-default approach to transition
+restrictions. By default, all implied transitions are allowed. The
+set of allowed transitions may be reduced by adding explicit transition
+restrictions in the `road.restrictions.turns` property.
+
+Turn restrictions come in two flavors: simple and via. A simple turn
+restriction allows a simple regulation to be stated, such as "No right
+turn onto Elm Street". A via restriction covers more elaborate cases
+where the sequence of maneuvers is important.
+
+<Tabs>
+  <TabItem value="simple" label="Simple restriction" default>
+
+<div style={{ width: "40.5%", float: "left" }}>
+
+<figure>
+
+![A simple transition restriction prohibiting a right turn](/img/transportation/turn-restriction-01.svg)
+
+<figcaption>
+
+*Prohibited right turn from "source" to "target" segment at connector 2.*
+
+</figcaption>
+
+</figure>
+
+</div>
+
+<div style={{ width: "59.5%", float: "right" }}>
+
+<details>
+<summary>Source segment</summary>
+<CodeBlock language="yaml">{ ExampleTurnRestriction1Source }</CodeBlock>
+</details>
+
+<details>
+<summary>Connector 1</summary>
+
+*This connector is not an important part of the example, since it does
+not participate in the turn restriction, but it is included to bring
+real-world context to the example.*
+
+<CodeBlock language="yaml">{ ExampleTurnRestriction1Connector1 }</CodeBlock>
+</details>
+
+<details>
+<summary>Exit segment</summary>
+
+*This segment is not an important part of the example, since all implied
+transitions are allowed on it. We include it to bring real-world context
+to the example.*
+
+<CodeBlock language="yaml">{ ExampleTurnRestriction1Exit }</CodeBlock>
+
+</details>
+
+<details>
+<summary>Connector 2</summary>
+
+*The right turn from the source segment (`forward` heading) to the
+target segment (`forward` heading) is prohibited at this connector.*
+
+<CodeBlock language="yaml">{ ExampleTurnRestriction1Connector2 }</CodeBlock>
+</details>
+
+<details>
+<summary>Target segment</summary>
+
+*Traffic heading `forward` on the source segment may not enter this
+segment heading `forward`, i.e. the right turn from the source segment
+to this segment is prohibited.*
+
+<CodeBlock language="yaml">{ ExampleTurnRestriction1Source }</CodeBlock>
```

Comment:
the codeblock should refer to ExampleTurnRestriction1Target rather than ExampleTurnRestriction1Source

Reply:
Fixed. Thanks for catching this issue!